### ISSUE TITLE
increased/decreased copulatory plug formation

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3508]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. decreased copulatory plug formation
2. increased copulatory plug formation

>2. Synonyms (e.g., absent spleen)

1. decreased mating plug formation ;  decreased vaginal plug formation
2. increased mating plug formation ;  increased vaginal plug formation

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1.  decrease in the number deposition of plugs of sperm or other gelatinous material in the opening of the vaginas in a study population of females by a male(s) at the termination of copulation
2.  increase in the number deposition of plugs of sperm or other gelatinous material into the opening of the vaginas in a study population of females by a male(s) at the termination of copulation

PMID:33368416

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal copulatory plug formation (MP:0030982)
2. abnormal copulatory plug formation (MP:0030982)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0003-3645-6803

**Comment by anna-anagnostop** [comment]
CHANGE DEF: decrease in the number of copulatory plugs deposited in a study population of females by a male(s) at the termination of copulation

**Comment by anna-anagnostop** [comment]
Added new terms:
decreased copulatory plug deposition MP:0031436
increased copulatory plug deposition MP:0031437
